source/framework/analysis/src/TRestRealTimeDrawingProcess.cxx

```diff
@@ -254,7 +254,7 @@ void TRestRealTimeDrawingProcess::DrawWithNotification() {
                 int _runNumber = run->GetRunNumber();
                 delete run;
                 if (_runNumber == runNumber) {
-                    for (int i = 0; i < fPlots.size(); i++) {
+                    for (unsigned int i = 0; i < fPlots.size(); i++) {
                         fPlots[i]->SetFile(message);
                         fPlots[i]->PlotCombinedCanvas();
                     }
```

Comment:
In general we should try to refactor similar statements whenever we find the time (just a reminder).

```suggestion
                    for (auto& plot: fPlots) {
                        plot->SetFile(message);
                        plot->PlotCombinedCanvas();
                    }
```